src/sentry/static/sentry/app/views/projectFilters.jsx

```diff
@@ -297,19 +309,58 @@ const ProjectFiltersSettingsForm = React.createClass({
     );
   },
 
+  renderAdditionalFilters() {
+    let errors = this.state.errors;
+    return (
+      <div>
+        <h5>{t('Filter errors from these releases:')}</h5>
+        <TextareaField
+          key="release"
+          name="release"
+          help={this.renderLinkToGlobWiki()}
+          placeholder="e.g. 1.* or [!3].[0-9].*"
+          value={this.state.formData['filters:releases']}
+          error={errors['filters:releases']}
+          onChange={this.onFieldChange.bind(this, 'filters:releases')}
+        />
+        <h5>{t('Filter errors by error message:')}</h5>
+        <TextareaField
+          key="errorMessage"
+          name="errorMessage"
+          help={this.renderLinkToGlobWiki()}
+          placeholder="e.g. TypeError* or *: integer division or modulo by zero"
+          value={this.state.formData['filters:error_messages']}
+          error={errors['filters:error_messages']}
+          onChange={this.onFieldChange.bind(this, 'filters:error_messages')}
+        />
+      </div>
+    );
+  },
+
+  renderDisabledFeature() {
+    let project = this.getProject();
+    let organization = this.getOrganization();
+    return this.state.hooksDisabled
+      .map(hook => {
```

Comment:
nit: can be more concise with
```javascript
.map(hook => hook(organization, project))
```

Reply:
👍

Reply:
Although if I change it, it won't matter.